dissect/target/volume.py

```diff
@@ -138,10 +161,35 @@ class LogicalVolumeSystem(VolumeSystem):
     logical volume systems.
     """
 
+    @classmethod
+    def detect_volume(cls, fh: BinaryIO) -> bool:
+        """Determine whether the given file-like object belongs to this logical volume system.
+
+        The position of ``fh`` will be restored before returning.
+
+        Args:
+            fh: A file-like object that may be part of the logical volume system.
+
+        Returns:
+            ``True`` if the given file-like object is part of the logical volume system, ``False`` otherwise.
+        """
+        offset = fh.tell()
+        try:
+            return cls._detect_volume(fh)
```

Comment:
Shouldn't it do an `fh.seek(0)` first? Otherwise the docstring of `_detect_volume()` is incorrect.

And I noiticed not all `detect_volume()`'s originally did this, that won't be a problem?

Reply:
Nope, the places where it didn't do it eventually did it somewhere in the underlying implementation. It's just nicer to ensure it from now on.